subsys/bluetooth/audio/csip_set_coordinator.c

```diff
@@ -1575,8 +1575,8 @@ static int bt_csip_set_coordinator_get_lock_state(
 		 * so we can just initiate the ordered access procedure (oap) callback directly
 		 * here.
 		 */
-
-		if (!active.oap_cb(active.info, active.members, active.members_count)) {
+		if (!active.oap_cb ||
```

Comment:
```suggestion
		if (active.oap_cb == NULL ||
```

I prefer the explicit NULL checks

Reply:
I think you forgot this one :)